### PR TITLE
fix: update Sei symbol to redEth

### PR DESCRIPTION
**Pull request by ltyu**

### CODE REVIEW

File: deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml [hunk 1]
```diff
@@ -239,4 +239,4 @@ tokens:
     logoURI: /deployments/warp_routes/EZETH/logo.svg
```
Comment (paulbalaji) [inline deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml:239]: diff logo too?

File: deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml [hunk 2]
```diff
@@ -239,4 +239,4 @@ tokens:
...
     name: Renzo Restaked ETH
```
Comment (paulbalaji) [inline deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml:240]: same name or diff name?

File: deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml (hunk 1 above)
Comment (ltyu) [reply deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml]: new logo. added

File: deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml (hunk 2 above)
Comment (ltyu) [reply deployments/warp_routes/EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit-config.yaml:240]: same name for now

File: deployments/warp_routes/EZETH/RedETH-logo.svg [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          <path class="cls-4" d="M287.96,393.29l-1.58,1.93v98.41l1.39,4.06c.08.23.39.27.53.08l127.97-180.23-128.31,75.76Z"/>
+          <path class="cls-3" d="M287.96,497.67v-104.38l-128.24-75.76,127.91,180.25c.1.14.33.07.33-.1Z"/>
+          <path class="cls-4" d="M288.68,368.59l126.87-74.99c.27-.16.25-.56-.04-.7l-127.55-57.98v133.26c0,.37.4.59.71.41Z"/>
+          <path class="cls-3" d="M160.29,293.54l126.98,75.06c.3.18.69-.04.69-.39v-133.29l-127.64,58.02c-.25.11-.27.47-.03.61Z"/>
+        </g>
+        <polyline class="cls-1" points="159.73 305.1 287.96 381.15 416.27 305.1"/>
+      </g>
+    </g>
+  </g>
+</svg>
```
Comment (paulbalaji) [inline deployments/warp_routes/EZETH/RedETH-logo.svg:52]: ran through svg minifier
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 576"><defs><style>.cls-3,.cls-4{stroke-miterlimit:10;stroke:#ee2e2c}.cls-3{fill:#700f10}.cls-4{fill:#4c1211}</style></defs><g id="Layer_1"><path d="m288 80-3 10v276l3 3 128-76L288 80z" style="stroke:#ee2e2c;fill:#7f1417;stroke-miterlimit:10"/><path d="M288 80 160 293l128 76V80Z" style="fill:#9f2a29;stroke:#ee2e2c;stroke-miterlimit:10"/><path d="m288 393-2 2v99l2 4 128-180-128 75Z" class="cls-4"/><path d="M288 498V393l-128-75 128 180Z" class="cls-3"/><path d="m289 369 127-75v-1l-128-58v133l1 1Z" class="cls-4"/><path d="m160 294 127 75 1-1V235l-128 58v1Z" class="cls-3"/><path d="m160 305 128 76 128-76" style="stroke-miterlimit:10;fill:none;stroke:#7f1417;stroke-width:4px"/></g></svg>
```

**Pull request merged**